domains/jk.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "jack",
+    "email": "15014586591@139.com"
+  },
+  "record": {
+     "AAAA": [
+      "2a00:dcc0:1ea5:5ca9:0000:0000:6301:cc4a"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.